design/news/news-design.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  "views": {
+    "by_section_and_id_and_date": {
+      map: function (doc) {
+        if (doc.docType !== 'message' || !Array.isArray(doc.viewIn)) { return; }
+
+        payload = {
```

Comment:
Should be a `var` here (CouchDB design docs use older JS syntax before `let` and `const`). Seems like it's working without it, but we should have that in there to avoid unintended behavior.